core/Cargo.toml

```diff
@@ -295,14 +295,15 @@ sha2 = { version = "0.10", optional = true }
 sled = { version = "0.34.7", optional = true }
 suppaftp = { version = "5.2", default-features = false, features = [
   "async-secure",
+  "rustls",
   "async-rustls",
-  "async-native-tls",
 ], optional = true }
 tikv-client = { version = "0.3.0", optional = true, default-features = false }
 tokio = "1.27"
 tokio-postgres = { version = "0.7.8", optional = true }
 tracing = { version = "0.1", optional = true }
 uuid = { version = "1", features = ["serde", "v4"] }
+async-tls = { version = "0.12.0", optional = true }
```

Comment:
And this?

Reply:
Rust-tls was not support async TLSConnector in the default implementation, So we need async-tls in here, Or we can use https://github.com/rustls/tokio-rustls

Reply:
Actually, suppaftp also use `async-tls`
https://github.com/privacy-com/suppaftp/blob/4217710fb457e679411ca128b0f2ed07e1850456/suppaftp/src/async_ftp/tls/rustls.rs#L9

Reply:
https://github.com/veeso/suppaftp/blob/3251558dfd66ece468da46e700a8f8f245dcd6e4/suppaftp/Cargo.toml#L57

Reply:
Got it. I thought we don't depend on it directly.